src/workspaces.c

```diff
@@ -319,6 +350,11 @@ workspaces_switch_to(struct workspace *target, bool update_focus)
 	lab_cosmic_workspace_set_hidden(
 		server->workspaces.current->cosmic_workspace, true);
 
+	lab_ext_workspace_set_active(
+		server->workspaces.current->ext_workspace, false);
+	lab_ext_workspace_set_hidden(
+		server->workspaces.current->ext_workspace, true);
```

Comment:
The integration of both, cosmic and ext are likely using the the `hidden` flag wrong. It appears to mean something like `disabled` instead which is a concept labwc doesn't support. We should likely simply remove both calls.